site/layouts/shortcodes/yamlEvents.html

```diff
@@ -0,0 +1,31 @@
+<table>
```

Comment:
Should be rendered as a `<ul>` instead. We can change that in a separate PR.

Reply:
I can change the shortcode to render as an unordered list - is that what we would prefer to a table ?

Reply:
yes, I think it would be easier to read. Happy to make the change after you merge.

Reply:
Lets look at that as a separate PR, I had a play with ul and couldn't get it to look good